.changeset/pretty-wolves-kiss.md

```diff
@@ -0,0 +1,7 @@
+---
+"kilo-code": minor
+---
+
+Adds support for changing the Kilo Code providers routing strategy
+
+You can now select the OpenRouter provider to process your Kilo Code requests.
```

Comment:
We could specifically call out Kimi K2, because there the endpoint selection matters a lot for price and context window.